libnopegl/src/utils.h

```diff
@@ -109,6 +109,7 @@
                          NGLI_ARG_VEC4((v)+4*2),    \
                          NGLI_ARG_VEC4((v)+4*3)
 
+#define NGLI_HAS_FLAGS(a, b) (((a) & (b)) == (b))
```

Comment:
Kind of annoying to change that in your history but I think `NGLI_HAS_ALL_FLAGS` would be more explicit. Feel free to ignore.

Reply:
Renamed to ALL_FLAGS.